modules/ocf_desktop/files/xsession/Xsession

```diff
@@ -114,7 +114,10 @@ xset m 0 0
 sleep 8 && /usr/local/bin/fix-audio &
 
 # Auto logout inactive users
-/usr/bin/xautolock -locker /usr/local/bin/auto-lock -time 7 -noclose &
+# killer is called killtime minutes after locker is called
+# any user input including shaking the mouse then not clearing the lockscreen is considered user input
```

Comment:
I just think that 90 minutes would miss pretty much all the cases we care about, like when the lab is open, and would only cover cases where OCF staff stay logged in overnight or something.